troposphere/s3.py

```diff
@@ -46,6 +46,23 @@ class VersioningConfiguration(AWSProperty):
     }
 
 
+class AccelerateConfiguration(AWSProperty):
+    props = {
+        'AccelerationStatus': (basestring, True),
+    }
+    acceleration_status_values = [
+        'Enabled',
+        'Suspended',
+    ]
+
+    def validate(self):
```

Comment:
I'd remove this, and instead write a custom validator here that does the `acceleration_status_values`:

```
def validate_acceleration_status(value):
    valid_status = ["Enabled", "Suspended"]
    if value not in valid_status:
        raise ValidationError("AccelerationStatus must be one of: %s", ", ".join(valid_status))
    return value
```

The benefit of this is that it will take into account non-string values, such as Functions like `Ref()` (say someone wanted to pass this as a Parameter, for example, that'd be a `Ref`).

Then you can just use that validator in place of `basestring`.